src/main/webapp/app/course/manage/course-lti-configuration/edit-course-lti-configuration.component.ts

```diff
@@ -34,6 +45,13 @@ export class EditCourseLtiConfigurationComponent implements OnInit {
         private ltiConfigurationService: LtiConfigurationService,
     ) {}
 
+    handleScroll = (): void => {
```

Comment:
Question: What is the reason for implementing an auto-scroll instead of the otherwise standard paged list? You also added such a list to the other component.

If there is a good reason, consider extracting this functionality to avoid such generic functionality specialisation in this specific component.

Reply:
The auto-scroll feature was initially implemented nicer usage in dropdown menu. The other component does not use a dropdown for listing LTI platforms, which is why it did not include this feature. Based on your feedback that the auto-scroll is not essential, I have removed this functionality to maintain consistency with the other component.